src/dom/react-nodegui/src/components/Svg/RNSvg.ts

```diff
@@ -0,0 +1,67 @@
+import { setViewProps, ViewProps, WidgetEventListeners } from "../View/RNView";
+import {
+  QSvgWidget,
+  NodeWidget,
+  QWidgetSignals,
+  CursorShape,
+  QCursor,
+  QBoxLayoutSignals,
+  QFont,
+  QGraphicsEffect,
+  NodeLayout,
+  Component,
+} from "@nodegui/nodegui";
+import { RNComponent, RNProps } from "../config";
+import { throwUnsupported } from "../../utils/helpers";
+
+export interface SvgProps extends ViewProps<QBoxLayoutSignals> {
+  /**
+   * Sets the graphics effect used by the widget
+   */
+  graphicsEffect?: QGraphicsEffect<any>;
+
+  /**
+   * Sets the file path
+   */
+  src?: string | Buffer;
+}
+
+const setSvgProps = (
+  widget: RNSvg,
+  newProps: SvgProps,
+  oldProps: SvgProps
+) => {
+  const setter: SvgProps = {
+    set graphicsEffect(effect: QGraphicsEffect<any>) {
+      widget.setGraphicsEffect(effect);
+    },
+    set src(file: string | Buffer) {
+      widget.load(file);
+    },
+  };
+  Object.assign(setter, newProps);
+  setViewProps(widget, newProps, oldProps);
+};
+
+/**
+ * @ignore
+ */
+export class RNSvg extends QSvgWidget implements RNComponent {
+  static tagName = "qtsvg";
```

Comment:
I've checked this myself and it's fine as just "svg" after all – no conflicts with the Svelte language tooling.
```suggestion
  static tagName = "svg";
```